packages/babel-plugin-jest-hoist/src/__tests__/babel-plugin-jest-hoist-test.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const path = require('path');
+const babel = require('babel-core');
+
+const rootDir = path.resolve(__dirname);
+const options = {plugins: [rootDir + '/../index']};
+
+describe('babel-plugin-jest-hoist', () => {
+
+  // Read all test dirs
+  fs.readdirSync(rootDir).forEach(suiteName => {
+    const filePath = rootDir + '/' + suiteName;
+
+    if (fs.statSync(filePath).isDirectory()) {
+
+      // Run test case with fixtures
+      it(suiteName.replace(/-/g, ' '), () => {
+        const expected = fs.readFileSync(filePath + '/expected.js').toString();
+        const actual = fs.readFileSync(filePath + '/actual.js').toString();
```

Comment:
Can this use the new snapshot feature in Jest 14 instead? It save you having to manually load/update your fixtures.